Puzzle/src/org/fundationjala/devops/bootcamp/main/java/Game.java

```diff
@@ -0,0 +1,97 @@
+import java.util.Scanner;
+
+public class Game {
+    Scanner sc = new Scanner(System.in);
+    public void run() {
+        System.out.println("Welcome to the game!");
+        System.out.print("Set size of the board: ");
+        int limit = sc.nextInt();
+        Table table = new Table(limit);
+        Movement movement = new Movement();
+
+        System.out.println("Table created");
+        table.printTable();
+
+
+        setHoleInSpecificPosition(table, movement);
+
+        play(table, movement);
+
+    }
+
+    private void play(Table table, Movement movement) {
```

Comment:
In this method, you are initializing the menu options before starting playing, maybe you can rename this method